pydra/engine/specs.py

```diff
@@ -141,9 +141,11 @@ def __getattr__(self, name):
     def __repr__(self):
         return "LF('{0}', '{1}')".format(self.node.name, self.field)
 
-    def get_value(self):
+    def get_value(self, name=None):
+        if name is None:
```

Comment:
i don't think this is necessary.  we don't want an argument for this function.

Reply:
ok

Reply:
but it's still not clear why you only set `self.field` in `__getattr__`

Reply:
because you don't know what the field is when nodebase `getattr` is called. 

when `x=task.lzout.field` is made

`NodeBase` `gettattr` returns a `LazyField` object, on which `getattr` is called about `field` and that's when we set it and return the `LazyField` instance.

Reply:
ok, I thought that maybe it should be set when `get_value` and that's why I added `name`